Plugins/Wox.Plugin.Program/Programs/Win32.cs

```diff
@@ -133,7 +140,9 @@ public List<Result> ContextMenus(IPublicAPI api)
                     Title = api.GetTranslation("wox_plugin_program_open_containing_folder"),
                     Action = _ =>
                     {
-                        Main.StartProcess(Process.Start, new ProcessStartInfo(ParentDirectory));
+
+
+                        Main.StartProcess(Process.Start, new ProcessStartInfo("explorer", ParentDirectory));
```

Comment:
nit:  If your referencing the same process in multiple places in the application this could be a constant could be in its own .cs class.

Reply:
I tried to change the core code of wox as little as possible, only upgrading what was deprecated in .net core.

Reply:
> I tried to change the core code of wox as little as possible, only upgrading what was deprecated in .net core.

I wasn't thinking of anything major,  Just a class `ProcessNames` that has a const string member `Explorer`.  This could be referenced everywhere.  It's a pretty nit comment, but this allows you to have a cohesive and central place that identifies the processes being executed by wox, and avoids typos in string literals.